packs/consul/actions/register_service.py

```diff
@@ -0,0 +1,9 @@
+from lib import action
+
+
+class ConsulRegisterServiceAction(action.ConsulBaseAction):
+    def run(self, node, service, address, port, tags, dc):
+
+        definition = {"Service": service, "Port": port, "Tags": tags}
+        result = self.consul.catalog.register(node, address, service=definition, dc=dc)
+        return (result, "")
```

Comment:
Actually, I missed this earlier - any particular reason you return a tuple?

You could simply just return a result.